src/http/api/account.ts

```diff
@@ -42,57 +49,49 @@ export function createGetAccountHandler(accountService: AccountService) {
     return async function handleGetAccount(ctx: AppContext) {
         const logger = ctx.get('logger');
         const site = ctx.get('site');
+        let account: AccountType | null = null;
+        const db = ctx.get('db');
 
-        // Validate input
-        const handle = ctx.req.param('handle') || '';
+        const apCtx = fedify.createContext(ctx.req.raw as Request, {
+            db,
+            globaldb: ctx.get('globaldb'),
+            logger,
+        });
 
-        if (handle === '') {
-            return new Response(null, { status: 400 });
+        const defaultAccount =
+            await accountService.getDefaultAccountForSite(site);
+
+        const handle = ctx.req.query('handle');
```

Comment:
Curious - why did we move to a query param here? I think the existing pattern is route params, not a big deal I think but I'd like to understand why

Reply:
Earlier in the account API the handle was always passed as 'index' and it assumed that it needs to return the profile of the current user. 

I changed it to :-

Getting profile for myself -> `GET /.ghost/activitypub/account`
Getting profile for others -> `GET /.ghost/activitypub/account/?handle=@username@domain`

My thinking is that the handle is optional so it should be in a query param. 

Other approach could be to keep sending 'index' for fetching "My profile" and have a check in backend.

Reply:
Changed it to route params to make consistent with other endpoints